cheatsheets/Content_Security_Policy_Cheat_Sheet.md

```diff
@@ -8,10 +8,15 @@ The increase in XSS and clickjacking vulnerabilities demands a more `defense in
 
 # Avoid CSP
 
+As mentioned in the [w3c specifications](https://www.w3.org/TR/CSP3/#intro):
+> CSP is not intended as a first line of defense against content injection vulnerabilities. Instead, CSP is best used as defense-in-depth. It reduces the harm that a malicious injection can cause, but it is not a replacement for careful input validation and output encoding.
+
 If you are a developer of any of the applications mentioned below, CSP will barely provide or improve their security:
-- Single page applications with no cookies or authentication.
+- Single page applications with no cookies or authentication and that serve static content (the application does not use input to generate content).
 - Applications that are already vulnerable to XSS vulnerabilities and chose not to remediate them. CSP is not the first line of defense.
 
+*Note:* Despite CSP not being a first line of defense, using CSP to protect the user inside the browser from a vulnerability that is not going to be fixed or is under work **is recommended**.
```

Comment:
and most importantly as a second line of defense when the primary security mechanisms fail i.e. to mitigate vulnerabilities that are not know yet.